src/main/java/ch/njol/skript/conditions/CondCompare.java

```diff
@@ -83,7 +82,9 @@ public class CondCompare extends Condition {
 			{"(1¦neither|) %objects@-1% (was|were) [(equal to|the same as)] %objects%", Relation.EQUAL},
 			{"(1¦neither|) %objects@-1% (was|were) between %objects% and %objects%", Relation.EQUAL},
 			{"(1¦neither|) %objects@-1% (2¦)(was not|were not|wasn't|weren't) between %objects% and %objects%", Relation.EQUAL},
-			
+			{"(1¦neither|) %objects@-1% (was|were) (caused|done|made) by %objects%", Relation.EQUAL},
```

Comment:
I don't like this because this is (as far as I can see) the only place this relation would be used, and every other event it _could_ be used in wouldn't be a correct use of it.
I think part of the problem is that it's only coincidentally true because of a language quirk (i.e. when you say the `damage was caused by fire` the relation is actually saying the damage cause is equal to the fire damage cause, but it doesn't look like that) but every other place with past time states where the relation can be used doesn't have this quirk and so just looks wrong.

Reply:
I feel like the best approach (if you need this to be a compare condition) would be to make it its own kind of relation condition that we could apply selectively, but then it might be more accurately applied to events generally (e.g. the player's death was caused by ...) and that would create its own mess.